packages/shorebird_cli/lib/src/commands/run_command.dart

```diff
@@ -24,6 +24,12 @@ class RunCommand extends ShorebirdCommand
         abbr: 't',
         help: 'The main entrypoint file of the application.',
       )
+      // ignore: lines_longer_than_80_chars
+      ..addOption(
```

Comment:
This should be a multi-option and then you don't need to split on commas etc, `package:args` handles it for you.

```suggestion
      ..addMultiOption(
```

Reply:
Thank you for this! I def missed to read the docs throughly 😓 , have amended the changes!

Reply:
One comment on this, we build our apps with package:args as well, and some of our dart defines have commas in them. 

We set `splitCommas: false` on `addMultiOption` to support this, otherwise it splits on that dart-define entry's comma. I don't think dart-define supports setting values using comma separation anyway (?), so I don't think setting this to false would break anything.